py-polars/polars/interchange/buffer.py

```diff
@@ -0,0 +1,20 @@
+import enum
+from typing import Tuple
+
+# TODO: no numpy dep, pyarrow/pa.Array?
```

Comment:
Should the `Buffer` class use pyarrow here? From what I can tell it doesn't implement the `__array_interface__`. So `Buffer.ptr` needs to be rolled here and I'm guessing that includes handling when arrays are chunked.